docs/articles/sdk/configure-qtcreator.rst

```diff
@@ -0,0 +1,59 @@
+Configuring QtCreator to use the SDK
+====================================
+
+After the installation of the SDK and ``source``ing the environment, there are couple of
+steps to do to run QtCreator in the SDK environment. This roadmap assumes
+that SDK and QtCreator are installed and the environment is ``source``d already.
+
+Adding a new kit
+--------------------
+
+To be able to cross-compile via QtCreator, a new build kit should be added to
```

Comment:
You don't really add a kit to QtCreator, only to this setup. Just rephase it to "should be added."

Reply:
Not sure if i get it correctly but done some change